ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -1405,6 +1406,61 @@ func TestConvertToTaskDefinitionWithECSParamsVolumeWithoutNameError(t *testing.T
 	assert.Error(t, err, "Expected error converting Task Definition with ECS Params volume without name")
 }
 
+func TestConvertToTaskDefinitionWithEFSVolume(t *testing.T) {
```

Comment:
can we add tests for the error cases as well? (no system id, encryption required but missing)